chart/skywalking/templates/es-init.job.yaml

```
@@ -15,7 +15,7 @@
 
 # https://docs.sentry.io/server/installation/docker/#running-migrations
 
-{{- if hasPrefix "elasticsearch" .Values.oap.storageType }}
+{{- if hasSuffix "elasticsearch" .Values.oap.storageType }}
```

```suggestion
{{- if contains "elasticsearch" .Values.oap.storageType }}
```

@kezhenxu94 Could we apply this directly?